apps/contacts/service.py

```diff
@@ -48,3 +49,29 @@ def on_deleted(self, doc):
         :return:
         """
         push_notification('contacts:deleted', _id=[doc.get(config.ID_FIELD)])
+
+
+class OrganisationService(Service):
+    def get(self, req, lookup):
+        """
+        Search for organisation matching the passed q parameter
+        :param req:
+        :param lookup:
+        :return: List of matching organisation strings
+        """
+        new_req = ParsedRequest()
+
+        q_str = 'organisation:' + '* organisation:'.join(req.args.get('q', '').split()) + '*'
```

Comment:
is it supposed to be `organisation:* organisation:<q>*` ? I think [match_phrase_query](https://www.elastic.co/guide/en/elasticsearch/reference/2.4/query-dsl-match-query.html#query-dsl-match-query-phrase-prefix) could be better for the prefix search

Reply:
The idea is to wildcard each term. Not strictly a phrase search as I'm trying to match say "insurance australia" or "australian insurance". Each term is prefixed with "organisation:" An alternative might have been to set the default field to 'organisation'

Reply:
I see, makes sense then